ext/pdo_firebird/tests/testdb.inc

```diff
@@ -6,24 +6,16 @@
 // Password:	phpfi
 // Database:	<nothing>
 //
-// A path to a database must be specified by using PDO_FIREBIRD_TEST_DATABASE
+// A DSN must be specified by using PDO_FIREBIRD_TEST_DSN
 
-define('PDO_FIREBIRD_TEST_USERNAME', getenv('PDO_FIREBIRD_TEST_USERNAME') ?: 'SYSDBA');
-define('PDO_FIREBIRD_TEST_PASSWORD', getenv('PDO_FIREBIRD_TEST_PASSWORD') ?: 'phpfi');
-define('PDO_FIREBIRD_TEST_HOSTNAME', getenv('PDO_FIREBIRD_TEST_HOSTNAME') ?: 'localhost');
-define('PDO_FIREBIRD_TEST_DATABASE', getenv('PDO_FIREBIRD_TEST_DATABASE') ?: '');
+define('PDO_FIREBIRD_TEST_USER', getenv('PDO_FIREBIRD_TEST_USER') ?: 'SYSDBA');
```

Comment:
How about using the following code snippets:

```php
......
 getenv('PDO_FIREBIRD_TEST_USER') ?? 'SYSDBA';
......
```
It's the PHP 7 syntax.

Reply:
@peter279k we can’t as getenv() returns false, not null

Reply:
Ok, using the `?:` syntax to detect is false.